.github/workflows/bench.yml

```diff
@@ -43,43 +43,3 @@ jobs:
     - name: Detect Changes
       uses: './.github/actions/diffs'
       id: diff
-
-  indexer-bench:
```

Comment:
It seems that indexer-bench is the only thing run in this file so we can probably remove this file entirely?